packages/shylu/shylu_node/tacho/src/impl/Tacho_NumericTools_LevelSet.hpp

```diff
@@ -2827,7 +2839,7 @@ class NumericToolsLevelSet : public NumericToolsBase<ValueType, DeviceType> {
       auto vecX = (lvl%2 == 0 ? vecU : vecW);
       auto vecY = (lvl%2 == 0 ? vecW : vecU);
       status =
-        #if ROCM_VERSION >= 50400
+        #if (ROCM_VERSION >= 50400 && ROCM_VERSION < 60000)
```

Comment:
Looks like this could be made more consistent with similar changes by swapping the order of the check and function api, e.g. change to:
```
      #if (ROCM_VERSION >= 60000)
      rocsparse_spmv
      #else
      rocsparse_spmv_ex
```

Reply:
Thank you, @ndellingwood !! Yea, my checks are ugly. I was trying to reduce the code lines.. With ``ROCM_VERSION >= 60000``, it went back to an older naming of ``spmv`` (same as  ``ROCM_VERSION < 50400``) but with the same argument list as ``ROCM_VERSION >= 50400``. Maybe, I'll try defining the function name based on ROCM_VERSION on the top.